diff --git a/node_modules/react-native-worklets/android/CMakeLists.txt b/node_modules/react-native-worklets/android/CMakeLists.txt
index 1d949d6..739f4db 100644
--- a/node_modules/react-native-worklets/android/CMakeLists.txt
+++ b/node_modules/react-native-worklets/android/CMakeLists.txt
@@ -117,7 +117,7 @@ if(${JS_RUNTIME} STREQUAL "hermes")
   endif()
 elseif(${JS_RUNTIME} STREQUAL "jsc")
   if(ReactAndroid_VERSION_MINOR GREATER_EQUAL 76)
-    target_link_libraries(worklets ReactAndroid::jsctooling)
+    target_link_libraries(worklets ReactAndroid::reactnative)
   else()
     target_link_libraries(worklets ReactAndroid::jscexecutor)
   endif()
